stock_picking_return_lot/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+This module propagates serial numbers or lots form initial picking to backorder.
```

Comment:
This module propagates serial numbers or lots from initial picking to return picking.